`export-dynamic` command in quickstart is causing build failures, this is also propagated to 1.10  PR as well. we need to find and fix this issue in both the PRs.


```
ERROR  red-hat-developer-hub-backstage-plugin-quickstart#export-dynamic: command (/opt/app-root/src/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-quickstart) /tmp/xfs-12fa8dfe/yarn run export-dynamic exited (1)
 ERROR  run failed: command  exited (1)
```


I checked out this PR and did a `yarn export dynamic` and this is the exact error
```

Error: Failed to compile '../../node_modules/@red-hat-developer-hub/backstage-plugin-quickstart/dist/components/QuickstartContent/LightspeedIcon.esm.js':
  Module not found: Error: Can't resolve '../../images/lightspeed-icon-black.svg' in '/Users/kjeeyar/Documents/rhdh/dynamic-plugins/node_modules/@red-hat-developer-hub/backstage-plugin-quickstart/dist/components/QuickstartContent'
```

--- a/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-adoption-insights-backend-dynamic/package.json
+++ b/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-adoption-insights-backend-dynamic/package.json
@@ -1,6 +1,6 @@
 {
   "name": "red-hat-developer-hub-backstage-plugin-adoption-insights-backend",
-  "version": "0.8.1",
+  "version": "0.8.2",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
@@ -38,8 +38,8 @@
     "export-dynamic:clean": "run export-dynamic --clean"
   },
   "dependencies": {
-    "@red-hat-developer-hub/backstage-plugin-adoption-insights-backend": "0.8.1",
-    "@red-hat-developer-hub/backstage-plugin-adoption-insights-common": "0.8.1"
+    "@red-hat-developer-hub/backstage-plugin-adoption-insights-backend": "0.8.2",
+    "@red-hat-developer-hub/backstage-plugin-adoption-insights-common": "0.8.2"
   },
   "devDependencies": {
     "@backstage/cli": "0.36.0",

--- a/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-adoption-insights/package.json
+++ b/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-adoption-insights/package.json
@@ -1,6 +1,6 @@
 {
   "name": "red-hat-developer-hub-backstage-plugin-adoption-insights",
-  "version": "0.8.1",
+  "version": "0.8.2",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "@mui/material": "5.18.0",
-    "@red-hat-developer-hub/backstage-plugin-adoption-insights": "0.8.1"
+    "@red-hat-developer-hub/backstage-plugin-adoption-insights": "0.8.2"
   },
   "devDependencies": {
     "@backstage/cli": "0.36.0",

--- a/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-analytics-module-adoption-insights-dynamic/package.json
+++ b/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-analytics-module-adoption-insights-dynamic/package.json
@@ -1,6 +1,6 @@
 {
   "name": "red-hat-developer-hub-backstage-plugin-analytics-module-adoption-insights",
-  "version": "0.8.1",
+  "version": "0.8.2",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
@@ -42,8 +42,8 @@
     }
   },
   "dependencies": {
-    "@red-hat-developer-hub/backstage-plugin-adoption-insights-common": "0.8.1",
-    "@red-hat-developer-hub/backstage-plugin-analytics-module-adoption-insights": "0.8.1"
+    "@red-hat-developer-hub/backstage-plugin-adoption-insights-common": "0.8.2",
+    "@red-hat-developer-hub/backstage-plugin-analytics-module-adoption-insights": "0.8.2"
   },
   "devDependencies": {
     "@backstage/cli": "0.36.0",

--- a/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-bulk-import-backend-dynamic/package.json
+++ b/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-bulk-import-backend-dynamic/package.json
@@ -1,6 +1,6 @@
 {
   "name": "red-hat-developer-hub-backstage-plugin-bulk-import-backend",
-  "version": "7.3.4",
+  "version": "7.3.5",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
@@ -39,7 +39,7 @@
     "export-dynamic:clean": "run export-dynamic --clean"
   },
   "dependencies": {
-    "@red-hat-developer-hub/backstage-plugin-bulk-import-backend": "7.3.4"
+    "@red-hat-developer-hub/backstage-plugin-bulk-import-backend": "7.3.5"
   },
   "devDependencies": {
     "@backstage/cli": "0.36.0",

--- a/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-bulk-import/package.json
+++ b/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-bulk-import/package.json
@@ -1,6 +1,6 @@
 {
   "name": "red-hat-developer-hub-backstage-plugin-bulk-import",
-  "version": "7.3.4",
+  "version": "7.3.5",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
@@ -43,7 +43,7 @@
   },
   "dependencies": {
     "@mui/material": "5.18.0",
-    "@red-hat-developer-hub/backstage-plugin-bulk-import": "7.3.4"
+    "@red-hat-developer-hub/backstage-plugin-bulk-import": "7.3.5"
   },
   "devDependencies": {
     "@backstage/cli": "0.36.0",

--- a/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-global-floating-action-button/package.json
+++ b/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-global-floating-action-button/package.json
@@ -1,6 +1,6 @@
 {
   "name": "red-hat-developer-hub-backstage-plugin-global-floating-action-button",
-  "version": "1.9.2",
+  "version": "1.9.3",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@mui/material": "5.18.0",
-    "@red-hat-developer-hub/backstage-plugin-global-floating-action-button": "1.9.2"
+    "@red-hat-developer-hub/backstage-plugin-global-floating-action-button": "1.9.3"
   },
   "devDependencies": {
     "@backstage/cli": "0.36.0",

--- a/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-global-header/package.json
+++ b/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-global-header/package.json
@@ -1,6 +1,6 @@
 {
   "name": "red-hat-developer-hub-backstage-plugin-global-header",
-  "version": "1.21.4",
+  "version": "1.21.5",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@mui/material": "5.18.0",
-    "@red-hat-developer-hub/backstage-plugin-global-header": "1.21.4"
+    "@red-hat-developer-hub/backstage-plugin-global-header": "1.21.5"
   },
   "devDependencies": {
     "@backstage/cli": "0.36.0",

--- a/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-quickstart/package.json
+++ b/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-quickstart/package.json
@@ -1,6 +1,6 @@
 {
   "name": "red-hat-developer-hub-backstage-plugin-quickstart",
-  "version": "1.9.5",
+  "version": "1.9.6",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "@mui/material": "5.18.0",
-    "@red-hat-developer-hub/backstage-plugin-quickstart": "1.9.5"
+    "@red-hat-developer-hub/backstage-plugin-quickstart": "1.9.6"
   },
   "devDependencies": {
     "@backstage/cli": "0.36.0",

--- a/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-quickstart/package.json
+++ b/dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-quickstart/package.json
@@ -1,6 +1,6 @@
 {
   "name": "red-hat-developer-hub-backstage-plugin-quickstart",
-  "version": "1.9.4",
+  "version": "1.9.5",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "@mui/material": "5.18.0",
-    "@red-hat-developer-hub/backstage-plugin-quickstart": "1.9.4"
+    "@red-hat-developer-hub/backstage-plugin-quickstart": "1.9.5"
   },
   "devDependencies": {
     "@backstage/cli": "0.36.0",

